ttyclock.c

```diff
@@ -679,11 +678,89 @@ main(int argc, char **argv)
           update_hour();
           draw_clock();
           key_event();
+          ttyclock.lt = timestamp();
```

Comment:
maybe initializing it earlier would get rid of this seemingly stray instruction as well?